rotkehlchen/premium/premium.py

```diff
@@ -176,10 +178,13 @@ def __init__(self, credentials: PremiumCredentials):
             allowed_methods=False,  # retry on all method verbs
         )
         self.session.mount('https://', adapter)
+        self.session.verify = False
```

Comment:
what is this? Add a comment

Reply:
testing, forgot to remove. is for requests to avoid the ssl verification since the certificate exists but is not in any registry